src/libraries/System.Runtime.Serialization.BinaryFormat/src/Infos/ArrayInfo.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.IO;
+
+namespace System.Runtime.Serialization.BinaryFormat;
+
+/// <summary>
+///  Array information structure.
+/// </summary>
+/// <remarks>
+///  <para>
+///   <see href="https://learn.microsoft.com/openspecs/windows_protocols/ms-nrbf/8fac763f-e46d-43a1-b360-80eb83d2c5fb">
+///    [MS-NRBF] 2.4.2.1
+///   </see>
+///  </para>
```

Comment:
This reads as if you want the output to read "See [MS-NRBF] 2.4.2.1.".  I'm not sure what the doc generator / and/or docs site do with `<see href`, but I don't think they introduce the word See; otherwise a remark like `... for use with the <see cref="SomeType" /> type` would have a spurious "See" in the middle.

Reply:
It renders as a link without "See". My main motivation here is just to have a clickable link to the spec (I don't have any specific preferences).

![image](https://github.com/dotnet/runtime/assets/6011991/bf8b0977-574e-4aaf-a4a8-e0752361544a)

Reply:
Using [MetadataReader](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.metadata.metadatareader?view=net-8.0#remarks) as the basis of an example, this should probably be something like

> ArrayInfo records are described in \[MS-NRBF] 2.4.2.1.

Turning it into a sentence, rather than just a link.

Reply:
![image](https://github.com/dotnet/runtime/assets/6011991/37e66a38-768b-46b1-b595-2c1e6944a6b4)